.travis.yml

```diff
@@ -73,5 +73,8 @@ matrix:
   - os: linux
     python: '3.5'
     env: T=py35
+  - os: linux
+    python: '3.8'
+    env: T=py38
```

Comment:
Feel free to drop Python 3.5 from this config, too.  That was from an earlier failed porting effort.  We don't need to keep trying to support it.